pkg/logger/logger.go

```diff
@@ -36,8 +60,41 @@ func InitLog(log *logr.Logger) {
 	Log = *log
 }
 
-// If debug set in configmap, set log level to 1.
-// If loglevel set in command line and greater than debug log level, set it to command line level.
+type FilteredCore struct {
+	zapcore.Core
+	filter func(zapcore.Entry) bool
+}
+
+func (c *FilteredCore) Check(entry zapcore.Entry, ce *zapcore.CheckedEntry) *zapcore.CheckedEntry {
+	if c.filter(entry) {
+		return ce
+	}
+	return c.Core.Check(entry, ce)
+}
+
+func (c *FilteredCore) Write(entry zapcore.Entry, fields []zapcore.Field) error {
+	if c.filter(entry) {
+		return nil
+	}
+	return c.Core.Write(entry, fields)
+}
+
+func (c *FilteredCore) With(fields []zapcore.Field) zapcore.Core {
+	return &FilteredCore{
+		Core:   c.Core.With(fields),
+		filter: c.filter,
+	}
+}
+
+func filterRecorderLogs(entry zapcore.Entry) bool {
+	if strings.Contains(entry.Caller.File, "recorder/recorder.go") {
+		return true
+	}
+	return false
+}
+
+// If debug set in configmap, set the log level to 1.
+// If loglevel set in the command line and greater than the debug log level, set it to command line level.
 func getLogLevel(cfDebug bool, cfLogLevel int) int {
 	logLevel := 0
```

Comment:
when log level was info, you need to show Warning level log.

Reply:
refer latest version